main_with_gui.py

```diff
@@ -202,14 +228,14 @@ def __init__(self, window: QMainWindow, has_dedicated_gpu: bool):
 
 		self.setLayout(v_box)
 
-	def extract_data_from_generated_files(self, window: QMainWindow, has_dedicated_gpu: bool):
+	def extract_data_from_generated_files(self, window: QMainWindow, has_dedicated_gpu: bool, gpu_in_cpu: bool):
 		try:
 			if DEBUG_DIR:
 				files_dir = DEBUG_DIR
 			else:
 				files_dir = "tmp"
 			system_info, print_lspci_lines_in_dialog = extract_and_collect_data_from_generated_files(files_dir,
-				has_dedicated_gpu, False)  # TODO: support this
+				has_dedicated_gpu, gpu_in_cpu)  # TODO: support this
```

Comment:
We can finally remove this comment 🎉
```suggestion
				has_dedicated_gpu, gpu_in_cpu)
```